src/core/sign.ml

```diff
@@ -236,8 +238,17 @@ let add_symbol : t -> expo -> prop -> match_strat -> strloc -> term ->
     ; sym_expo ; sym_tree = ref Tree_types.empty_dtree
     ; sym_mstrat = ref sym_mstrat }
   in
+  sign.sign_syntax := SymMap.add sym Unqual !(sign.sign_syntax);
   sign.sign_symbols := StrMap.add s.elt (sym, s.pos) !(sign.sign_symbols); sym
 
+(** [strip_private sign] removes private symbols from signature [sign]. *)
+let strip_private : t -> unit = fun sign ->
```

Comment:
I cannot see where it is used.

Reply:
In `Compile.compile`. I thought better to move these operations in `Sign` because they access the internals of the `sign` datatype.